bingo/symbolic_regression/symbolic_regressor.py

```diff
@@ -157,136 +223,63 @@ def fit(self, X, y, sample_weight=None):
             print("sample weight not None, TODO")
```

Comment:
Are we planning to ever implement sample weighting?

Reply:
seems useful, but perhaps a lower priority until someone really needs it